src/resources/views/crud/fields/rate.blade.php

```diff
@@ -0,0 +1,67 @@
+<!-- radio -->
+@php
+    $value = old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : 0 ));
+    $field['count'] = $field['count'] ?? 5;
+@endphp
+
+@include('crud::fields.inc.wrapper_start')
+    <div>
+        <label>{!! $field['label'] !!}</label>
+        @include('crud::fields.inc.translatable_icon')
+    </div>
+
+    <div class="rate form-check">
+        @while ($field['count']--)
+            <input @include('crud::fields.inc.attributes') type="radio" id="star{{ $field['count'] }}" name="{{$field['name']}}" value="{{ $field['count']+1 }}" {{ $field['count']+1 == $value ? 'checked' : '' }}/>
+            <label for="star{{ $field['count'] }}" title="{{ $field['count']+1 }}">{{ $field['count'] }} stars</label>
+        @endwhile
+    </div>
+
+    {{-- HINT --}}
+    @if (isset($field['hint']))
+        <p class="help-block" style="clear: both;">{!! $field['hint'] !!}</p>
+    @endif
+@include('crud::fields.inc.wrapper_end')
+
+@if ($crud->fieldTypeNotLoaded($field))
+    {{-- FIELD SPECIAL CSS  --}}
```

Comment:
You should probably call ```markFieldTypeAsLoaded()``` first thing inside the IF statement. That way the CSS is not loaded twice. Take a look at this example here - https://github.com/Laravel-Backpack/CRUD/blob/master/src/resources/views/crud/fields/ckeditor.blade.php